servers/services/src/main/java/org/projectnessie/services/authz/AccessChecker.java

```diff
@@ -16,135 +16,129 @@
 package org.projectnessie.services.authz;
 
 import java.security.AccessControlException;
+import java.util.Map;
 import org.projectnessie.model.Content;
 import org.projectnessie.model.ContentKey;
 import org.projectnessie.versioned.NamedRef;
 
 /**
- * The purpose of the {@link AccessChecker} is to make sure that a particular role is allowed to
- * perform an action (such as creation, deletion) on a {@link NamedRef} (Branch/Tag). Additionally,
- * this interface also provides checks based on a given {@link ContentKey}.
+ * Accept various allowance-checks and retrieve the result of all operations at once.
+ *
+ * <p>The purpose of the {@link AccessChecker} is to accept all required checks via the {@code
+ * can...()} methods and return the result of these "can do xyz" checks via {@link #check()}.
+ *
+ * <p>The checks make sure that a particular role is allowed to perform an action (such as creation,
+ * deletion) on a {@link NamedRef} (Branch/Tag). Additionally, this interface also provides checks
+ * based on a given {@link ContentKey}.
  */
 public interface AccessChecker {
```

Comment:
updates in this file also need to be reflected in the markdown file for authorization